templates/credentials/your-exams.html

```diff
@@ -3,175 +3,189 @@
 {% block title %}Canonical Credentials -- Your exams{% endblock %}
 
 {% block meta_description %}
-    The Canonical Ubuntu Essentials exams certify knowledge and verify skills in general Linux,
-    Ubuntu Desktop, and Ubuntu Server topics.
+  The Canonical Ubuntu Essentials exams certify knowledge and verify skills in general Linux,
+  Ubuntu Desktop, and Ubuntu Server topics.
 {% endblock meta_description %}
 
 {% block meta_copydoc %}
-    https://docs.google.com/document/d/1QuhO-9FEOGLrYp8bErS_9snqdljl7d6tFAUoNQxoVDQ/edit
+  https://docs.google.com/document/d/1QuhO-9FEOGLrYp8bErS_9snqdljl7d6tFAUoNQxoVDQ/edit
 {% endblock
 meta_copydoc %}
 
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
 {% block content %}
 
-    <section class="p-strip">
-        <div class="row--50-50">
-            <div class="col">
-                <h1>
-                    <strong>Your exams</strong>
-                </h1>
-            </div>
-            <div class="col">
-                <p>
-                    The table below shows the exam attempts you have activated.
-                    <br />
-                    To take an exam, you need to schedule it and log back in during your booked time slot.
-                </p>
-                <a class="p-button" href="/credentials/redeem">Redeem an exam</a>
-            </div>
+  <section class="p-strip u-no-padding--bottom">
+    <div class="row--50-50">
+      <div class="col">
+        <h1>
+          <strong>Your exams</strong>
+        </h1>
+      </div>
+      <div class="col">
+        <p>The table below shows the exam attempts you have activated.</p>
+        <p>To take an exam, you need to schedule it and log back in during your booked time&nbsp;slot.</p>
+        <a class="p-button" href="/credentials/redeem">Redeem an exam</a>
+      </div>
+    </div>
+  </section>
+  <div class="p-suru"></div>
```

Comment:
You can use `p-suru` (or `p-suru--25-75`, to be more exact about the variant) as a wrapper around content, instead of the strip (with padding removed) you have on top.

Reply:
```suggestion
```